presto-native-execution/presto_cpp/main/types/VeloxToPrestoExpr.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "presto_cpp/external/json/nlohmann/json.hpp"
+#include "presto_cpp/presto_protocol/presto_protocol.h"
+#include "velox/core/Expressions.h"
+#include "velox/serializers/PrestoSerializer.h"
+
+using namespace facebook::velox;
```

Comment:
Velox guidelines disallow using namespace directives in header files. Please specify the fully qualified names below.